### ISSUE TITLE
`no-non-function-verb-prefix`: add ignore option

### ISSUE THREAD

**Issue by FloEdelmann** [sindresorhus/eslint-plugin-unicorn#3468]
### Description

`addOns` is reported by the rule (even though it is a noun). We use it several times in our project, so inline-disable comments are not viable. I suggest adding an `ignore` option to the rule, so we can keep valid reports for other variables starting with `add`.

### Examples

```js
// with "unicorn/no-non-function-verb-prefix": ["error", { "ignore": ["addOns"] }]

// ❌
const addPizza = new Pizza();

// ✅
const addedPizza = new Pizza();
const addOns = ["cheese", "pineapple"]
```
